SUMMARY.md

```diff
@@ -6,132 +6,132 @@
-  - [Error Handling](handbook/technical-overview/api-design-guide/errors.md)
-  - [Example API Service](handbook/technical-overview/api-design-guide/example.md)
-  - [GraphQL Naming Conventions](handbook/technical-overview/api-design-guide/graphql-naming-conventions.md)
-  - [HTTP Status Response Codes](handbook/technical-overview/api-design-guide/http-status-codes.md)
-  - [Methods](handbook/technical-overview/api-design-guide/methods.md)
-  - [Naming Conventions](handbook/technical-overview/api-design-guide/naming-conventions.md)
-  - [Once Only Principle](handbook/technical-overview/api-design-guide/once-only.md)
-  - [Resource Oriented Design](handbook/technical-overview/api-design-guide/resource-oriented-design.md)
-  - [Security](handbook/technical-overview/api-design-guide/security.md)
-  - [Versioning](handbook/technical-overview/api-design-guide/versioning.md)
-  - [Kerfislýsingar](handbook/technical-overview/auth/System.md)
-    - [Service Provider Manual](handbook/technical-overview/auth/system/service-provider-manual/README.md)
-      - [Audience](handbook/technical-overview/auth/system/service-provider-manual/audience.md)
-      - [the Authentication Service Provision](handbook/technical-overview/auth/system/service-provider-manual/authentication-service-provision.md)
-      - [Incidents](handbook/technical-overview/auth/system/service-provider-manual/incidents.md)
-      - [Intervention by Code Change](handbook/technical-overview/auth/system/service-provider-manual/intervention-by-code-change.md)
-      - [Trouble Shooting](handbook/technical-overview/auth/system/service-provider-manual/trouble-shooting.md)
-    - [System Manual](handbook/technical-overview/auth/system/system-manual/README.md)
-      - [Introduction](handbook/technical-overview/auth/system/system-manual/introduction.md)
-      - [Service Provider’s Role](handbook/technical-overview/auth/system/system-manual/service-provider-role.md)
-- [Code Reviews](handbook/technical-overview/code-reviews.md)
-- [Code Standards](handbook/technical-overview/code-standards.md)
-- [Devops](handbook/technical-overview/devops/README.md)
-  - [Continuous Delivery](handbook/technical-overview/devops/continuous-delivery.md)
-  - [Dockerizing](handbook/technical-overview/devops/dockerizing.md)
-  - [Environment Setup](handbook/technical-overview/devops/environment-setup.md)
-  - [Logging](handbook/technical-overview/devops/logging.md)
-  - [Metrics](handbook/technical-overview/devops/metrics.md)
-  - [Observability](handbook/technical-overview/devops/observability.md)
-  - [Operations Base Principles](handbook/technical-overview/devops/operations-base-principles.md)
-- [Monorepo](handbook/technical-overview/monorepo.md)
-- [Personas](handbook/technical-overview/personas.md)
-- [Teamwork](handbook/technical-overview/teamwork.md)
-- [Technical Direction](handbook/technical-overview/technical-direction.md)
+- [Overview](handbook\technical-overview\README.md)
```

Comment:
Looks like all the indents got lost. Did you run `yarn gitbook`? Could it be a prettier configuration remove the indents?

Reply:
backslash vs. forwardslash?

How does gitbook handle that?

Reply:
I imagine it will break because it's not a valid relative path, isn't it?